packages/playground/package.json

```diff
@@ -75,6 +72,9 @@
     "cross-env": "~7.0.3",
     "typescript": "~4.7.2",
     "@types/lz-string": "~1.3.34",
-    "@types/debounce": "~1.2.1"
+    "@types/debounce": "~1.2.1",
+    "debounce": "~1.2.1",
```

Comment:
Just `@types/swagger-ui` should be a devDependencies the other should stay as `dependencies`